### PR TITLE
Adding conda-replicate

### PR DESCRIPTION
**Pull request by cswartzvi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cswartzvi** [comment]
Source: https://github.com/cswartzvi/conda-replicate

**Comment by cswartzvi** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by cswartzvi** [comment]
@conda-forge-admin, please ping team

File: recipes/conda-replicate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - pydantic
+    - requests
+    - pyyaml
+
+test:
+  imports:
+    - {{ package_name }}
+
+  commands:
+    - {{ name }} --help
```
Comment (ocefpaf) [inline recipes/conda-replicate/meta.yaml:45]: The jinja here obfuscate the recipe and doesn't really save that much typing. I'd rather see the form of the import and the CLI invocations.

Comment (cswartzvi) [reply recipes/conda-replicate/meta.yaml]: I was a little over-zealous with the templating. I have specified the import and CLI invocation directly.

File: recipes/conda-replicate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  url: https://github.com/cswartzvi/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5432e410cecd5826df55a8ccafa056ae6c13508b13db3ef032182cd4b262e6e4
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - {{ name|lower }} = conda_replicate.cli:app
```
[the remark this answers is not in the document]
Comment (cswartzvi) [reply recipes/conda-replicate/meta.yaml]: I have removed the jinja templating in the entrypoint.

File: recipes/conda-replicate/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/conda-replicate/meta.yaml]: Yeah. It is tempting but for folks like me, who are constantly reading/modifying recipes, I prefer to use jinja vars only for the version or PyPI names. We rarely read the rendered recipe. We usually see the "raw" one on GH.

**Pull request merged**